site/src/pages/DeploySettingsPage/DeploySettingsLayout.tsx

```diff
@@ -52,19 +55,15 @@ const DeploySettingsLayoutInner: FC = () => {
         <Stack css={{ padding: "48px 0" }} direction="row" spacing={6}>
           <Sidebar />
           <main css={{ maxWidth: 800, width: "100%" }}>
-            {deploymentConfigQuery.data ? (
-              <DeploySettingsContext.Provider
-                value={{
-                  deploymentValues: deploymentConfigQuery.data,
-                }}
-              >
-                <Suspense fallback={<Loader />}>
-                  <Outlet />
-                </Suspense>
-              </DeploySettingsContext.Provider>
-            ) : (
-              <Loader />
-            )}
+            <DeploySettingsContext.Provider
```

Comment:
When checking if the data exists, we can ensure the children's pages can access the data directly without needing to handle the loading state. Removing this check requires each page to deal with the loading itself(adding an extra amount of work). Maybe I'm missing something but I don't see value in this change.

Reply:
Not all users coming to this page can access the deployment settings.  Some of them can only access the organization information or only the audit log, so if we add loading here then those users can never see the page as they will only see a loader.  So we have to handle the loader on each individual page since they all have kinds of permissions.

I think maybe we could do this differently by not combining different things into the same page, but that would require a rework of the design.

Reply:
An alternative would be to always run the query and just let it error for users without the right permissions.  We could then move the loader back here.  But we would still need to handle the error state on each page.  Or we could ignore the error state, since users should not be seeing links to those pages anyway since they lack permissions.

Reply:
> Not all users coming to this page can access the deployment settings. Some of them can only access the organization information or only the audit log or only the users etc, 

Makes sense. 👍